validation/oauth_config.xsd

```diff
@@ -34,6 +35,7 @@
       <xs:element type="xs:string" name="tokenUri"/>
       <xs:element type="xs:string" name="userInfoUri" maxOccurs="1" minOccurs="0"/>
       <xs:element type="xs:string" name="instanceUrlValidationRegex" maxOccurs="1" minOccurs="0"/>
+      <xs:element type="xs:string" name="defaultInstanceUrl" maxOccurs="1" minOccurs="0"/>
```

Comment:
Do we have an item to ensure the [connector-packager](https://github.com/tableau/connector-plugin-sdk/tree/master/connector-packager) sets `min-version-tableau` to 2023.1 if `defaultInstanceUrl` usage is detected?

Reply:
I think we are actually going to hold off on adding defaultInstanceUrl for now until we can do more testing on it.  Should I still do this if oauthConfigId is detected?